stripe-core/build.gradle

```diff
@@ -25,6 +25,7 @@ dependencies {
     implementation libs.kotlin.coroutines
     implementation libs.kotlin.coroutinesAndroid
     implementation libs.kotlin.serialization
+    implementation libs.androidx.activity
```

Comment:
Can you run `ruby scripts/update_transitive_dependencies.rb` to make align our dependencies baseline with this change.